.eslintignore

```diff
@@ -1 +1,3 @@
-coverage/
\ No newline at end of file
+coverage/
```

Comment:
I must say, I am thoroughly impressed with the level of detail and effort put into the PR description. The inclusion of video explanations truly sets it apart. Keep up the outstanding work! 💯🎉 Congratulations, your code has been approved! 🎉💯

Reply:
thank you @Rustamxon7 for the feedback.